frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -91,6 +99,10 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
   const version = getQueryArgument('version');
   const [updateVersion, setUpdateVersion] = React.useState(version || currentLatestVersion);
 
+  const tokenType = 'role ARN';
+  const tokenizedAuth = getQueryArgument('tokenizedAuth');
```

Comment:
Sorry.  I should have been more clear.  

Remove this line and lines 93 and 99 and line 89 becomes 
`const { catalogNamespace, channel, pkg, tokenizedAuth, version } = getURLSearchParams();`. This simply deduplicates the multiple `getQueryArgument` calls and consolidates them in to the existing `getURLSearchParams` one (which returns an object of parameters and not a single parameter).